CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We added a different background color to the search bar to indicate when the search syntax is wrong. [#11658](https://github.com/JabRef/jabref/pull/11658)
 - We added a setting which always adds the literal "Cited on pages" text before each JStyle citation. [#11691](https://github.com/JabRef/jabref/pull/11732)
 - We added a new plain citation parser that uses LLMs. [#11825](https://github.com/JabRef/jabref/issues/11825)
+- We added a welcome screen that displays when no database is open [#koppor96](https://github.com/koppor/jabref/issues/96)
```

Comment:
May need to change the format to: 
`[koppor#96](https://github.com/koppor/jabref/issues/96)`

Reply:
fixed via: https://github.com/HaynesCrossley-ANU/jabref/pull/2/commits/2e386359fa944965158f21310e8453b6c3a5225f